python-package/lightgbm/engine.py

```diff
@@ -131,12 +131,16 @@ def train(params, train_data, num_boost_round=100,
         predictor = init_model
     else:
         predictor = None
+    init_iteration = predictor.init_iteration if predictor else 0
```

Comment:
I think the usage is:
1. Can get num_iteration from Predictor.
2. Set Init_iteration=predictor.num_iteration in Booster.
3. loop iteration into [Init_iteration, Init_iteration+num_boost_round)